src/parameterizations/lateral/MOM_Zanna_Bolton.F90

```diff
@@ -15,13 +15,16 @@ module MOM_Zanna_Bolton
 use MOM_domains,       only : pass_var, CORNER
 use MOM_cpu_clock,     only : cpu_clock_id, cpu_clock_begin, cpu_clock_end
 use MOM_cpu_clock,     only : CLOCK_MODULE, CLOCK_ROUTINE
+use MOM_ANN,           only : ANN_init, ANN_apply, ANN_end, ANN_CS
 
 implicit none ; private
 
 #include <MOM_memory.h>
 
 public ZB2020_lateral_stress, ZB2020_init, ZB2020_end, ZB2020_copy_gradient_and_thickness
 
+logical, public :: true_vorticity !< Use correct curvilinear coordinate expression for vorticity
```

Comment:
We do not allow module data (because it is not thread safe) except for single-threaded debugging purposes. Since this is needed in the hor_visc module I suggest adding this logical to the `HOR_VISC_CS` type. I also suggest changing the parameter name to something less controversial since "true" is in the eye of the beholder. How about "USE_CIRCULATION__IN_STRAIN" ?